src/main/java/de/tum/cit/aet/artemis/lecture/web/LectureTranscriptionResource.java

```diff
@@ -0,0 +1,184 @@
+        LectureUnit lectureUnit = lectureUnitRepository.findByIdElseThrow(lectureUnitId);
+        if (!lectureUnit.getLecture().getId().equals(lectureId)) {
+            throw new BadRequestAlertException("Lecture unit does not belong to the specified lecture", "lectureTranscription", "lectureUnitDoesNotMatchLecture");
+        }
+
+        if (lectureUnit.getLectureTranscription() != null) {
+            lectureTranscriptionRepository.deleteById(lectureUnit.getLectureTranscription().getId());
+            lectureUnit.setLectureTranscription(null);
+        }
+
+        LectureTranscription lectureTranscription = new LectureTranscription(transcriptionDTO.language(), transcriptionDTO.segments(), lectureUnit);
+        lectureTranscription.setId(null);
+
+        LectureTranscription result = lectureTranscriptionRepository.save(lectureTranscription);
+
+        return ResponseEntity.created(new URI("/api/lecture/" + lectureId + "/transcriptions/" + result.getId())).body(result);
+    }
+
+    /**
+     * GET /lectures/:lectureId/transcriptions/:transcriptionId : get the transcription for the transcriptionId.
+     *
+     * @param lectureId       the lectureId of the lecture from the transcription
+     * @param transcriptionId the transcriptionId of the transcription to retrieve
+     * @return the ResponseEntity with status 200 (OK) and with body the transcription, or with status 404 (Not Found)
+     */
+    @GetMapping("{lectureId}/transcription/{transcriptionId}")
+    @EnforceAtLeastStudent
+    public ResponseEntity<LectureTranscription> getLectureTranscriptions(@PathVariable Long lectureId, @PathVariable Long transcriptionId) {
+        log.debug("REST request to get transcription {}", transcriptionId);
+        LectureTranscription transcription = lectureTranscriptionRepository.findById(transcriptionId).orElseThrow();
+        authCheckService.checkIsAllowedToSeeLectureElseThrow(transcription.getLectureUnit().getLecture(), userRepository.getUserWithGroupsAndAuthorities());
+
+        return ResponseEntity.ok(transcription);
+    }
+
+    /**
+     * POST lecture/{lectureId}/lecture-unit/{lectureUnitId/ingest-transcription
+     * This endpoint is for starting the ingestion of all lectures or only one lecture when triggered in Artemis.
+     *
+     * @param lectureId     The id of the lecture of the transcription
+     * @param lectureUnitId The id of the lectureUnit that should be ingested
+     * @return the ResponseEntity with status 200 (OK) and a message success or null if the operation failed
+     */
+    @Profile(PROFILE_IRIS)
+    @PutMapping("lectures/{lectureId}/lecture-unit/{lectureUnitId}/ingest-transcription")
+    @EnforceAtLeastInstructor
+    public ResponseEntity<Void> ingestTranscriptions(@PathVariable Long lectureId, @PathVariable Long lectureUnitId) {
+        Lecture lecture = lectureRepository.findById(lectureId).orElseThrow();
+        Course course = lecture.getCourse();
```

Comment:
We would need to prevent that instructors from other courses can access and modify resources of other courses for all endpoints with `@EnforceAtLeastInstructor`.

```suggestion
        User user = userRepository.getUserWithGroupsAndAuthorities();
        authCheckService.checkIsAllowedToSeeLectureElseThrow(lecture, user);
        Course course = lecture.getCourse();
```